_data/legal.yml

```diff
@@ -0,0 +1,278 @@
+
+    - name: Lawgeex
+      url: https://www.lawgeex.com/
+      img: lawgeex.png
+      tfa: No
+      twitter: Lawgeex_
+      facebook: lawgeex
+
+    - name: Lecorpio
+      url: https://www.lecorpio.com
+      img: lecorpio.png
+      tfa: No
+      twitter: Lecorpio
+      facebook: Lecorpio-103184026413756
+
+    - name: Legal Robot
+      url: https://www.legalrobot.com
+      img: legalrobot.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+
+    - name: Legalzoom
+      url: https://www.legalzoom.com/
+      img: legalzoom.png
+      tfa: No
+      twitter: LegalZoom
+      facebook: LegalZoom
+
+    - name: Leverton
+      url: https://leverton.ai/
+      img: leverton.png
+      tfa: No
+      twitter: LEVERTON
+      facebook: levertonbrand
+
+    - name: Lex Machina
+      url: https://lexmachina.com
+      img: lexmachina.png
+      tfa: No
+      twitter: LexMachina
+      facebook: lexmachinainc
+
+    - name: LexisNexis Legal
+      url: https://www.lexisnexis.com/legal/
+      img: lexisnexislegal.png
+      tfa: No
+      twitter: lexisnexislegal
+      facebook: lexisnexislegal
+
+    - name: Lexoo
+      url: https://www.lexoo.co.uk/
+      img: lexoo.png
+      tfa: No
+      twitter: Lexoo
+      facebook: lexoo.co.uk
+
+    - name: Luminance
+      url: https://www.luminance.com
+      img: luminance.png
+      tfa: No
+      twitter: LuminanceTech
+
+    - name: Ravel Law
+      url: http://ravellaw.com/
+      img: ravellaw.png
+      tfa: No
+      twitter: ravellaw
+      facebook: ravellaw
+
+    - name: RightSignature
+      url: https://rightsignature.com/
+      img: rightsignature.png
+      tfa: No
+      twitter: RightSignature
+      facebook: RightSignature
+
+    - name: RocketLawyer
+      url: https://www.rocketlawyer.com/
+      img: rocketlawyer.png
+      tfa: No
+      twitter: RocketLawyer
+      facebook: RocketLawyer
+
+    - name: Rocketmatter
+      url: https://www.rocketmatter.com/
+      img: rocketmatter.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.rocketmatter.com/technology/security-lawyers-two-factor-authentication-use/
+
+    - name: Ross Intelligence
+      url: http://www.rossintelligence.com/
+      img: rossintelligence.png
+      tfa: No
+      twitter: ROSSIntel
+      facebook: rossintelligence
+
+    - name: Seal Software
+      url: https://www.seal-software.com/
+      img: sealsoftware.png
+      tfa: No
+      twitter: SealSoftware
+      facebook: SealSoftware
+
+    - name: Shake Law
+      url: http://www.shakelaw.com/
+      img: shakelaw.png
+      tfa: No
+      twitter: ShakeLaw
+
+    - name: SignNow
+      url: https://www.signnow.com/
+      img: signnow.png
+      tfa: No
+      twitter: signnow
+      facebook: BarracudaSignNow
+
+    - name: SignRequest
+      url: https://signrequest.com
+      img: signrequest.png
+      tfa: No
+      twitter: SignRequest
+      facebook: SignRequest
+
+    - name: Spring CM
```

Comment:
Spring CM seems to offer solutions other than simply legal.

anyone else think may not be the best category?